devserver fails to start

```
[INFO] Launching Vert.x Application
[INFO] Vert.x application redeploy enabled
[INFO] No plugin execution collected. The vertx:initialize goal has not been run beforehand. Only handling resources and java compilation
[INFO] Observing path:/home/andrew/workspace/cryostat/src/main
Jan. 28, 2022 4:25:17 P.M. io.vertx.core.impl.launcher.commands.Watcher
INFO: Watched paths: [/home/andrew/workspace/cryostat/target/classes]
Jan. 28, 2022 4:25:17 P.M. io.vertx.core.impl.launcher.commands.Watcher
INFO: Starting the vert.x application in redeploy mode
Starting vert.x application...
0c1ae254-5b53-41b7-a29f-bbc449cb99b8-redeploy
Jan. 28, 2022 4:25:18 P.M. io.cryostat.core.log.Logger info
INFO: cryostat started.
Jan. 28, 2022 4:25:18 P.M. io.cryostat.core.log.Logger info
INFO: Selected NoSSL strategy
Jan. 28, 2022 4:25:18 P.M. io.cryostat.core.log.Logger warn
WARNING: No available SSL certificates. Fallback to plain HTTP.
Jan. 28, 2022 4:25:18 P.M. io.cryostat.core.log.Logger info
INFO: Local config path set as /tmp/tmp.EarFXG1Y2x/conf
Warning: Nashorn engine is planned to be removed from a future JDK release
Jan. 28, 2022 4:25:19 P.M. io.cryostat.core.log.Logger info
INFO: Selecting configured PlatformDetectionStrategy "io.cryostat.platform.internal.DefaultPlatformStrategy"
Jan. 28, 2022 4:25:19 P.M. io.cryostat.core.log.Logger info
INFO: Selected Default Platform Strategy
Jan. 28, 2022 4:25:19 P.M. io.cryostat.core.log.Logger info
INFO: Local save path for flight recordings set as /tmp/tmp.EarFXG1Y2x/archive
Jan. 28, 2022 4:25:19 P.M. io.cryostat.core.log.Logger info
INFO: HTTP service running on http://localhost:8181
Jan. 28, 2022 4:25:19 P.M. io.cryostat.core.log.Logger info
INFO: Selecting configured AuthManager "io.cryostat.net.NoopAuthManager"
Exception in thread "main" java.lang.RuntimeException: java.io.IOException: Probe template directory does not exist, must be set using environment variable CRYOSTAT_PROBE_TEMPLATE_PATH
	at io.cryostat.templates.TemplatesModule.provideLocalProbeTemplateService(TemplatesModule.java:69)
	at io.cryostat.templates.TemplatesModule_ProvideLocalProbeTemplateServiceFactory.provideLocalProbeTemplateService(TemplatesModule_ProvideLocalProbeTemplateServiceFactory.java:42)
	at io.cryostat.templates.TemplatesModule_ProvideLocalProbeTemplateServiceFactory.get(TemplatesModule_ProvideLocalProbeTemplateServiceFactory.java:32)
	at io.cryostat.templates.TemplatesModule_ProvideLocalProbeTemplateServiceFactory.get(TemplatesModule_ProvideLocalProbeTemplateServiceFactory.java:11)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
	at io.cryostat.net.web.http.api.beta.ProbeTemplateUploadHandler_Factory.get(ProbeTemplateUploadHandler_Factory.java:49)
	at io.cryostat.net.web.http.api.beta.ProbeTemplateUploadHandler_Factory.get(ProbeTemplateUploadHandler_Factory.java:13)
	at dagger.internal.SetFactory.get(SetFactory.java:126)
	at dagger.internal.SetFactory.get(SetFactory.java:37)
	at dagger.internal.DelegateFactory.get(DelegateFactory.java:36)
	at io.cryostat.net.web.WebModule_ProvideWebServerFactory.get(WebModule_ProvideWebServerFactory.java:50)
	at io.cryostat.net.web.WebModule_ProvideWebServerFactory.get(WebModule_ProvideWebServerFactory.java:15)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
	at dagger.internal.DelegateFactory.get(DelegateFactory.java:36)
	at io.cryostat.DaggerCryostat_Client.webServer(DaggerCryostat_Client.java:729)
	at io.cryostat.Cryostat.main(Cryostat.java:82)
Caused by: java.io.IOException: Probe template directory does not exist, must be set using environment variable CRYOSTAT_PROBE_TEMPLATE_PATH
	at io.cryostat.core.agent.LocalProbeTemplateService.<init>(LocalProbeTemplateService.java:69)
	at io.cryostat.templates.TemplatesModule.provideLocalProbeTemplateService(TemplatesModule.java:65)
	... 15 more
```

--- a/devserver.sh
+++ b/devserver.sh
@@ -6,7 +6,7 @@ if [ -z $MVN ]; then
     MVN=$(which mvn)
 fi
 
-for i in archive clientlib conf templates; do
+for i in archive clientlib conf templates probes; do
     if [ -e $work_dir/$i ]; then
         if [ ! -d $work_dir/$i ]; then
             echo "$work_dir/$i already exists but is not a directory"
@@ -38,4 +38,5 @@ MAVEN_OPTS="${flags[@]}" \
     CRYOSTAT_CLIENTLIB_PATH="$work_dir/clientlib" \
     CRYOSTAT_CONFIG_PATH="$work_dir/conf" \
     CRYOSTAT_TEMPLATE_PATH="$work_dir/templates" \
+    CRYOSTAT_PROBE_TEMPLATE_PATH="$work_dir/probes" \
     $MVN -Dcryostat.minimal=true clean vertx:run

